src/app/views/layouts/application.html.erb

```diff
@@ -6,6 +6,16 @@
     <%= stylesheet_link_tag 'application' %>
     <%= favicon_link_tag 'favicon.ico' %>
     <%= yield(:head) %>
+
+    <meta property="og:title" content="<%= yield(:title) || t('conference') %>" />
+    <meta name="twitter:title" content="<%= yield(:title) || t('conference') %>">
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:image" content="<%= image_url 'twitter-card.png' %>">
+    <% if meta_description %>
```

Comment:
Remind me again, an undefined local variable won’t throw an exception here, right? That is, we don’t need to first check is_defined? And then check if it’s set... probably a ruby 101 question that I should know. Thanks!

Reply:
I don't think it would ever be undefined, because it's defined in the application helper. So this `if` check probably isn't necessary and can be removed.